molecule/pdmysql/pdps/tasks/main.yml

```diff
@@ -99,13 +99,13 @@
     when: ansible_os_family == "RedHat"
```

Comment:
Please check pdps-update and pdps_setup and update them in scope of this PR respectively.